src/Common/src/Internal/Metadata/NativeFormat/Generator/NativeFormatGen.csproj

```diff
@@ -1,5 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
It is the eventual goal, but I do not plan to switch to the new project-system format as part of this change.